src/components/HURUmap/Tutorial/index.stories.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+import Tutorial from "@/pesayetu/components/HURUmap/Tutorial";
+import { tour } from "@/pesayetu/config";
+
+export default {
+  title: "Components/HURUmap/Tutorial",
+  argTypes: {},
+};
+
+const Template = ({ ...args }) => <Tutorial {...args} />;
+
+export const Default = Template.bind({});
+
+Default.args = tour;
```

Comment:
How do we use this component in storybook?

Reply:
resolved @kilemensi. thanks for pointing out